examples/allocator-client-csharp/README.md

```diff
@@ -0,0 +1,19 @@
+# A sample Allocator service C# client
```

Comment:
Should we link this somewhere, such as here? 
https://agones.dev/site/docs/advanced/allocator-service/#send-allocation-request

(and wrapped in a feature code block)

Reply:
Thanks for pointing it out.
PTAL. I added a note.